src/schemas/ev42/ev42_rw.cpp

```diff
@@ -151,20 +152,19 @@ HDFWriterModule::init_hdf(hdf5::node::Group &HDFGroup,
 
 HDFWriterModule::InitResult
 HDFWriterModule::reopen(hdf5::node::Group &HDFGroup) {
-  // Keep these for now, experimenting with those on another branch.
-  HDFIDStore *hdf_store = nullptr;
+  /// \todo Remove nullptr when clang-tidy merge is done.
```

Comment:
Are you keeping track of these todos in the jira ticket?

Reply:
In short: No.